### DISCUSSION TITLE
Show in the dashboard how long current subscription lasts

### DISCUSSION THREAD

**Discussion by pr0cop10** [simple-login/app#640] [Feature Requests]
Hi, I am currently subscribed to a premium plan. I want to request for the date and number of days until my subscription lasts to be shown in the dashboard.

The dashboard only says that I am on a premium plan.

Thanks.

**Comment by nguyenkims** [comment]
@pr0cop10 when you go to the Settings page, you can click on the "Manage Subscription" link which will show the expiration date of your subscription.
